share/man/nit.md

```diff
@@ -6,6 +6,8 @@ nit - interprets and debugs Nit programs.
 
 nit [*options*] FILE [ARG]...
 
+nit [*options*] - [ARG]...
```

Comment:
Using `-` in place of a path is standard. I should not add a synopsis line for that.